openblas: remove patch that's no longer needed (PR merged)

dtzWill opened the pull request:
As-is this fails to build since the patch doesn't apply.


<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

dtzWill committed:
openblas: remove patch that's no longer needed (PR merged)

matthewbauer commented:
This shouldn't be a mass rebuild right? Could you see if ```patches = []``` fixes this?

GrahamcOfBorg commented:
Success on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.41639&attempt_id=bbc21e4f-09a0-4650-ad74-bbd53483d639)

Attempted: openblas

<details><summary>Partial log (click to expand)</summary><p>

```
Install OK!
make[1]: Leaving directory '/build/OpenBLAS-0.3.0'
post-installation fixup
shrinking RPATHs of ELF executables and libraries in /nix/store/xb80dyv31df6fx8mdcxkivsh9m1kq8fh-openblas-0.3.0
shrinking /nix/store/xb80dyv31df6fx8mdcxkivsh9m1kq8fh-openblas-0.3.0/lib/libopenblas_armv8p-r0.3.0.dev.so
strip is /nix/store/3ki967c3ij4zi1q9ycc3g1vfwr608nrb-binutils-2.30/bin/strip
stripping (with command strip and flags -S) in /nix/store/xb80dyv31df6fx8mdcxkivsh9m1kq8fh-openblas-0.3.0/lib  /nix/store/xb80dyv31df6fx8mdcxkivsh9m1kq8fh-openblas-0.3.0/bin
patching script interpreter paths in /nix/store/xb80dyv31df6fx8mdcxkivsh9m1kq8fh-openblas-0.3.0
checking for references to /build in /nix/store/xb80dyv31df6fx8mdcxkivsh9m1kq8fh-openblas-0.3.0...
/nix/store/xb80dyv31df6fx8mdcxkivsh9m1kq8fh-openblas-0.3.0
```
</p></details>

GrahamcOfBorg commented:
Success on x86_64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.41639&attempt_id=07d9c93c-05d0-4eab-b5ee-72dea704607d)

Attempted: openblas

<details><summary>Partial log (click to expand)</summary><p>

```
Generating OpenBLASConfigVersion.cmake in /nix/store/pdv4gjsd4b5nlkh1k82xg0v8x3nww3kq-openblas-0.3.0/lib/cmake/openblas
Install OK!
make[1]: Leaving directory '/build/OpenBLAS-0.3.0'
post-installation fixup
shrinking RPATHs of ELF executables and libraries in /nix/store/pdv4gjsd4b5nlkh1k82xg0v8x3nww3kq-openblas-0.3.0
shrinking /nix/store/pdv4gjsd4b5nlkh1k82xg0v8x3nww3kq-openblas-0.3.0/lib/libopenblasp-r0.3.0.dev.so
strip is /nix/store/4mf2xm9p32lzrim927yk92xhx35yaz62-binutils-2.30/bin/strip
stripping (with command strip and flags -S) in /nix/store/pdv4gjsd4b5nlkh1k82xg0v8x3nww3kq-openblas-0.3.0/lib  /nix/store/pdv4gjsd4b5nlkh1k82xg0v8x3nww3kq-openblas-0.3.0/bin
patching script interpreter paths in /nix/store/pdv4gjsd4b5nlkh1k82xg0v8x3nww3kq-openblas-0.3.0
checking for references to /build in /nix/store/pdv4gjsd4b5nlkh1k82xg0v8x3nww3kq-openblas-0.3.0...
```
</p></details>

GrahamcOfBorg commented:
Failure on x86_64-darwin [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.41639&attempt_id=cb921d92-aa64-477e-bfd4-88b5bfb3dcbd)

Attempted: openblas

<details><summary>Partial log (click to expand)</summary><p>

```
clang-5.0: warning: argument unused during compilation: '-isystem /nix/store/j9wm8cmpszcyfcl8lsbn0j3zw1cxjx09-libc++-5.0.2/include' [-Wunused-command-line-argument]
clang-5.0: warning: argument unused during compilation: '-isystem /nix/store/j9wm8cmpszcyfcl8lsbn0j3zw1cxjx09-libc++-5.0.2/include' [-Wunused-command-line-argument]
clang-5.0: warning: argument unused during compilation: '-F/nix/store/jm3ac6lisvvlkqixxc4aih578w6makzx-CF-osx-10.10.5/Library/Frameworks' [-Wunused-command-line-argument]
clang-5.0: warning: argument unused during compilation: '-F/nix/store/jm3ac6lisvvlkqixxc4aih578w6makzx-CF-osx-10.10.5/Library/Frameworks' [-Wunused-command-line-argument]
gfortran -O2 -m128bit-long-double -Wall -m64 -fdefault-integer-8 -fPIC -c -o clahef_rk.o clahef_rk.f
gfortran -O2 -m128bit-long-double -Wall -m64 -fdefault-integer-8 -fPIC -c -o clahqr.o clahqr.f
f951: Warning: command line option '-Wformat=1' is valid for C/C++/ObjC/ObjC++ but not for Fortran
f951: Warning: command line option '-Wformat-security' is valid for C/C++/ObjC/ObjC++ but not for Fortran
building of '/nix/store/2qbmcg7dmmdh2b67val4jdacgmdp9ar0-openblas-0.3.0.drv' timed out after 1800 seconds
error: build of '/nix/store/2qbmcg7dmmdh2b67val4jdacgmdp9ar0-openblas-0.3.0.drv' failed
```
</p></details>

dtzWill committed:
openblas: workaround mass-rebuilds

Thanks for the suggestion:
https://github.com/NixOS/nixpkgs/pull/41639#issuecomment-395509669

GrahamcOfBorg commented:
Success on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.41639&attempt_id=548903c0-d759-49c2-8a8f-d99241ce0f09)

Attempted: openblas

<details><summary>Partial log (click to expand)</summary><p>

```
these paths will be fetched (1.97 MiB download, 10.57 MiB unpacked):
  /nix/store/7778440yqj6g3hcaq433i0zrc41ybyzk-openblas-0.3.0
copying path '/nix/store/7778440yqj6g3hcaq433i0zrc41ybyzk-openblas-0.3.0' from 'https://cache.nixos.org'...
/nix/store/7778440yqj6g3hcaq433i0zrc41ybyzk-openblas-0.3.0
```
</p></details>

GrahamcOfBorg commented:
Success on x86_64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.41639&attempt_id=7d3d5c13-c955-4533-943e-959ff7db0310)

Attempted: openblas

<details><summary>Partial log (click to expand)</summary><p>

```
these paths will be fetched (3.70 MiB download, 37.29 MiB unpacked):
  /nix/store/5c5ncg08ikzgxyqv2lfc5zqj7a9s5s4b-openblas-0.3.0
copying path '/nix/store/5c5ncg08ikzgxyqv2lfc5zqj7a9s5s4b-openblas-0.3.0' from 'https://cache.nixos.org'...
/nix/store/5c5ncg08ikzgxyqv2lfc5zqj7a9s5s4b-openblas-0.3.0
```
</p></details>

dtzWill commented:
@GrahamcOfBorg eval

(Is this how I ask ofborg to reconsider labels? O:))

GrahamcOfBorg commented:
Success on x86_64-darwin [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.41639&attempt_id=9fde9834-7c71-4ee8-87bf-d0e17200f411)

Attempted: openblas

<details><summary>Partial log (click to expand)</summary><p>

```
Generating openblas.pc in /nix/store/aybfrfn64a0661qwvwkjm5hdw2rg4bnb-openblas-0.3.0/lib/pkgconfig
Generating OpenBLASConfig.cmake in /nix/store/aybfrfn64a0661qwvwkjm5hdw2rg4bnb-openblas-0.3.0/lib/cmake/openblas
Generating OpenBLASConfigVersion.cmake in /nix/store/aybfrfn64a0661qwvwkjm5hdw2rg4bnb-openblas-0.3.0/lib/cmake/openblas
Install OK!
make[1]: Leaving directory '/private/tmp/nix-build-openblas-0.3.0.drv-0/OpenBLAS-0.3.0'
post-installation fixup
strip is /nix/store/8q05v40n0f7amgw70pk6bkb23r7hrzgn-cctools-binutils-darwin/bin/strip
stripping (with command strip and flags -S) in /nix/store/aybfrfn64a0661qwvwkjm5hdw2rg4bnb-openblas-0.3.0/lib  /nix/store/aybfrfn64a0661qwvwkjm5hdw2rg4bnb-openblas-0.3.0/bin
patching script interpreter paths in /nix/store/aybfrfn64a0661qwvwkjm5hdw2rg4bnb-openblas-0.3.0
/nix/store/aybfrfn64a0661qwvwkjm5hdw2rg4bnb-openblas-0.3.0
```
</p></details>

matthewbauer commented:
Oh yeah it's going to be a mass rebuild on Darwin no matter what because there's no glibc.

dtzWill commented:
Okay, great.  Just to double-check... yep here's the latest Hydra build: https://hydra.nixos.org/build/75603189/nixlog/1

GrahamcOfBorg commented:
Success on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.41639&attempt_id=f6a76378-5252-42d4-87b2-0845d8065a64)

Attempted: openblas

<details><summary>Partial log (click to expand)</summary><p>

```
/nix/store/7778440yqj6g3hcaq433i0zrc41ybyzk-openblas-0.3.0
```
</p></details>

GrahamcOfBorg commented:
Success on x86_64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.41639&attempt_id=cffd8a1a-4f4b-4fdf-9c2c-787e9c7c186a)

Attempted: openblas

<details><summary>Partial log (click to expand)</summary><p>

```
/nix/store/5c5ncg08ikzgxyqv2lfc5zqj7a9s5s4b-openblas-0.3.0
```
</p></details>

GrahamcOfBorg commented:
Success on x86_64-darwin [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.41639&attempt_id=ed07f955-d3df-4f77-8c4a-96e23a3a6a1b)

Attempted: openblas

<details><summary>Partial log (click to expand)</summary><p>

```
Generating openblas.pc in /nix/store/aybfrfn64a0661qwvwkjm5hdw2rg4bnb-openblas-0.3.0/lib/pkgconfig
Generating OpenBLASConfig.cmake in /nix/store/aybfrfn64a0661qwvwkjm5hdw2rg4bnb-openblas-0.3.0/lib/cmake/openblas
Generating OpenBLASConfigVersion.cmake in /nix/store/aybfrfn64a0661qwvwkjm5hdw2rg4bnb-openblas-0.3.0/lib/cmake/openblas
Install OK!
make[1]: Leaving directory '/private/tmp/nix-build-openblas-0.3.0.drv-0/OpenBLAS-0.3.0'
post-installation fixup
strip is /nix/store/8q05v40n0f7amgw70pk6bkb23r7hrzgn-cctools-binutils-darwin/bin/strip
stripping (with command strip and flags -S) in /nix/store/aybfrfn64a0661qwvwkjm5hdw2rg4bnb-openblas-0.3.0/lib  /nix/store/aybfrfn64a0661qwvwkjm5hdw2rg4bnb-openblas-0.3.0/bin
patching script interpreter paths in /nix/store/aybfrfn64a0661qwvwkjm5hdw2rg4bnb-openblas-0.3.0
/nix/store/aybfrfn64a0661qwvwkjm5hdw2rg4bnb-openblas-0.3.0
```
</p></details>
